src/components/font_editor/font_editor.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  Component,
+  onWillStart,
+  onWillUpdateProps,
+  useExternalListener,
+  useRef,
+  useState,
+} from "@odoo/owl";
+import { DEFAULT_FONT_SIZE, FONT_SIZES } from "../../constants";
+import { clip } from "../../helpers/index";
+import { setStyle } from "../../registries/index";
+import { SpreadsheetChildEnv, Style } from "../../types/index";
+import { css } from "../helpers/css";
+import { MenuState } from "../menu/menu";
+
+interface State {
+  menuState: MenuState;
+}
+
+interface Props {
+  callback: () => void;
+}
+
+// -----------------------------------------------------------------------------
+// TopBar
+// -----------------------------------------------------------------------------
+css/* scss */ `
+  .o-spreadsheet-topbar {
+    /* Toolbar + Cell Content */
+    .o-topbar-toolbar {
```

Comment:
Did not realise during my last review, sorry - the component should not be aware of its parents.
the style that concerns the compoment in inself (like moz/webkit  appreance shit) should be self contained but if some style is relying on the container (like topbar) then it should be transmitted through a props, be it as classes or specific style* .

  * your call - If some self-contained stuff can be expressed through bootstrap 5 classes I think you can go for it, otherwise, a style props might be in order.